### PR TITLE
typo fixes for coins and points

### PR DESCRIPTION
**Pull request by piyusharorawork**
Multiple occurences for **coin of** as well as **coin in** instead of **point of**  and **point in**  

 #551

### CODE REVIEW

**Comment by piyusharorawork** [comment]
@thesanjeevsharma can u look at this PR or create a separate issue for this .

File: src/containers/StyleGuide/StyleGuideCss/index.tsx [hunk 1]
```diff
@@ -258,7 +258,7 @@ const LeftMenu: FC = () => {
       </DocSubSection>
       <DocSubSection id={StyleGuideCssNav.blocksElements} title="Blocks vs. Elements">
         <p>
-          One of the pain-coins of <DocInlineCode>BEM</DocInlineCode> is the ambiguity of the distinction between Blocks
+          One of the pain-point of <DocInlineCode>BEM</DocInlineCode> is the ambiguity of the distinction between Blocks
```
Comment (thesanjeevsharma) [inline src/containers/StyleGuide/StyleGuideCss/index.tsx:261]: points*

**Comment by thesanjeevsharma** [comment]
@piyusharorawork Does this fix all the typos in docs? If not, can you have a look at it and make sure that all point/coin typos are fixed?

File: src/containers/StyleGuide/StyleGuideCss/index.tsx (hunk 1 above)
Comment (piyusharorawork) [reply src/containers/StyleGuide/StyleGuideCss/index.tsx]: fixed now

**Comment by piyusharorawork** [comment]
@thesanjeevsharma yes I went through all and looks fine to me now

**Comment by thesanjeevsharma** [comment]
Fixes #619

**Comment by Ayush-Porwal** [comment]
@thesanjeevsharma I made changes mentioned in the issue #619 let me know if you want me to create a pull create.

**Comment by thesanjeevsharma** [comment]
@Ayush-Porwal I'll merge this one. You can fix the remaining typos. I'll split the reward. 👍

**Comment by Ayush-Porwal** [comment]
@thesanjeevsharma I have created the pull request, please check it out https://github.com/thenewboston-developers/Website/pull/621

**Comment by thesanjeevsharma** [comment]
@piyusharorawork Can you tell me your account number so I can send you the coins for this PR?

**Comment by Ayush-Porwal** [comment]
@thesanjeevsharma do I get a reward too?

**Comment by piyusharorawork** [comment]
@thesanjeevsharma my account number in case 😄 
95a296a38cd77059a5d7330e65d9ee3b092e9e9e6b7b4a72b260c24f48024e03

**Pull request merged**